packages/devextreme/testing/tests/DevExpress.exporter/exceljsParts/exceljs.dataGrid.tests.js

```diff
@@ -40,8 +40,10 @@ const moduleConfig = {
     beforeEach: function() {
         this.worksheet = new ExcelJS.Workbook().addWorksheet('Test sheet');
         this.customizeCellCallCount = 0;
-        this.stub = sinon.stub(errors, 'log').callsFake(() => {
-            QUnit.assert.strictEqual(true, false, 'error.log should not be called');
+        this.stub = sinon.stub(errors, 'log').callsFake((errorId) => {
+            if(errorId === 'W0003') {
+                QUnit.assert.strictEqual(true, false, 'no deprecation warnings should be logged');
+            }
```

Comment:
Why did we change this test case?

Reply:
https://github.com/DevExpress/DevExtreme/pull/25510#discussion_r1327056916

Reply:
Sorry, I cannot agree with [this comment](https://github.com/DevExpress/DevExtreme/pull/25510#discussion_r1327056916).

Before this change, the test case checked the occurrence of any error.
But after, it will check only the 'W0003'.
This test case will pass if we have another deprecation warning with another code (or another warning).

It doesn't look okay, but maybe I missed something.
@ksercs, @pomahtri could you please provide an additional explanation for this change? 
(I mean is it okay or not from your point of view and why)

Reply:
Agree with @williamvinogradov, narrowing test case seems to be dangerous
Mocking checking license in tests looks most suitable, but code with `//#DEBUG` is fine too in case mocking in qunit is complex